config/gamesettings/default.xml

```diff
@@ -29,24 +29,27 @@
 		<broadcastPermissionPriceMod value="1.5"/>
 	</hard>
 
-	<!-- defaults, in case the actual levels do not want to define their own values -->
+	<!--
+		defaults, in case the actual levels do not want to define their own values.
+		If not statet otherwise, modifiers (...Mod) are factors, so 2 = double value and 0.5 = half the value
+	-->
 	<defaults>
-		<startMoney value="250000"/><!--owned money on game start-->
+		<startMoney value="250000"/><!--money owned on game start-->
 		<startCredit value="500000"/><!--credit from boss on game start;to be paid back-->
-		<creditMinimum value="600000"/><!--base credit available (depending on boss mood)-->
+		<creditMinimumOnGameStart value="600000"/><!--minimum credit available (depending on boss mood)-->
```

Comment:
Der Kommentar ist nicht korrekt.

Reply:
(knackige) Benennung und Kommentar sind wegen des Einfluss der Stimmung des Chefs nicht trivial. Der Wert ist 
weder ein echtes Minimum noch ein Maximum, da dieser Wert mit Werten zwischen 0.5 und 2 multipliziert wird. Der Wert spielt außerdem nicht nur beim Spielstart eine Rolle sondern ist ja auch der Basiswert für das gesamte weitere Spiel.
Der Kommentar vorher war wahrscheinlich schon besser und "creditAvailableBase" wäre ein besserer Name...

Reply:
Ich habe den Wert nochmal in creditAvailableOnGameStart umbenannt. Ich denke, das passt.